src/Adapter/Product/ProductStatusUpdater.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+
+namespace PrestaShop\PrestaShop\Adapter\Product;
```

Comment:
Miss header & declare strict_types

Reply:
Indeed, but nice idea to create a dedicated updater service for this